Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -124,6 +120,118 @@ public override void Serialize(IIonWriter writer, object item)
             writer.StepOut();
         }
 
+        private bool DeserializeProperty(PropertyInfo property, IIonReader reader, IonType ionType, ref object deserialized)
+        {
```

Comment:
Can you add a comment header for this method to mention the intent.

Reply:
Done.